### ISSUE TITLE
iOS's APP, when the keyboard show the toolbar will move out of the screen

### ISSUE THREAD

**Issue by Kary-Huang** [quilljs/quill#3185]
Please describe the a concise description and fill out the details below. It will help others efficiently understand your request and get to an answer instead of repeated back and forth. Providing a [minimal, complete and verifiable example](https://stackoverflow.com/help/mcve) will further increase your chances that someone can help.

**Steps for Reproduction**
**Expected behavior**: When the keyboard show, the toolbar should stick on the top

**Actual behavior**: The keyboard will move out of the screen, you can check the gif I posted.

**Platforms**: iOS

Include browser, operating system and respective versions

**Version**: 1.1.6

Run `Quill.version` to find out

![keyboard](https://user-images.githubusercontent.com/53503698/94538939-35afa780-0277-11eb-9f50-b109649a8aaa.gif)

**Comment by Kary-Huang** [comment]
Here is my HTML file

```
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport"
        content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
    <title>Title</title>

    <!-- Include the Quill library -->
    <script src="https://cdn.quilljs.com/1.1.6/quill.js"></script>
    <!-- Include Quill stylesheet -->
    <link href="https://cdn.quilljs.com/1.1.6/quill.snow.css" rel="stylesheet">
</head>

<body>
    <style>
        html,
        body {
            height: 100%;
            margin: 0;
            overflow: hidden;
        }

        #wrap {
            height: 100%;
            display: flex;
            flex-direction: column;
        }

        #editor {
            height: 100%;
            flex: 1;
            overflow: hidden;
        }

        .post-text-align-left {
            text-align: left;
        }

        .post-text-align-center {
            text-align: center;
        }

        .post-text-align-right {
            text-align: right;
        }

        @media (pointer: coarse) {
            .ql-snow.ql-toolbar button:hover:not(.ql-active),
            .ql-snow .ql-toolbar button:hover:not(.ql-active) {
                color: #444;
            }

            .ql-snow.ql-toolbar button:hover:not(.ql-active) .ql-fill,
            .ql-snow .ql-toolbar button:hover:not(.ql-active) .ql-fill,
            .ql-snow.ql-toolbar button:hover:not(.ql-active) .ql-stroke.ql-fill,
            .ql-snow .ql-toolbar button:hover:not(.ql-active) .ql-stroke.ql-fill {
                fill: #444;
            }

            .ql-snow.ql-toolbar button:hover:not(.ql-active) .ql-stroke,
            .ql-snow .ql-toolbar button:hover:not(.ql-active) .ql-stroke,
            .ql-snow.ql-toolbar button:hover:not(.ql-active) .ql-stroke-miter,
            .ql-snow .ql-toolbar button:hover:not(.ql-active) .ql-stroke-miter {
                stroke: #444;
            }
        }
    </style>

    <!-- Create the editor container -->
    <div id="wrap">
        <div id="editor"></div>
    </div>

    <!-- Initialize Quill editor -->
    <script>

        const Link = Quill.import('formats/link');
        Link.sanitize = function (url) {
            lowerCaseURL = url.toLowerCase()
            if (lowerCaseURL.startsWith('https')) {
                return lowerCaseURL
            } else {
                return 'https://' + lowerCaseURL
            }
        }

        const toolbarOptions = {
            container: [
                ['bold', 'italic', 'strike'],
                [{ 'list': 'ordered' }, { 'list': 'bullet' }],
                [{ header: [1, 2, false] }],
                ['link', 'image'],
                [{ align: '' }, { align: 'center' }, { align: 'right' }]
            ],
        }
        var editor = new Quill('#editor', {
            modules: {
                toolbar: toolbarOptions
            },
            theme: 'snow',
        })

        function setContent(html) {
            editor.root.innerHTML = html;
            setTimeout(function () {
                var length = editor.root.innerHTML.length;
                editor.setSelection(length, length);
            }
                , 20)
        }

        function getContent() {
            return editor.root.innerHTML
        }

        function clickImage() {
            webkit.messageHandlers.openAlbum.postMessage(null)
        }

        const toolbar = editor.getModule('toolbar')
        toolbar.addHandler('image', clickImage)
    </script>
</body>

</html>

```

**Comment by Kary-Huang** [comment]
@jhchen @luin 
And one more question, when I select `bold`, `italic`, `strike`, the text will go the next line after I tap return
You can check it base on my gif, thanks!

![keyboard2](https://user-images.githubusercontent.com/53503698/94547608-0fdbd000-0282-11eb-8905-59a6dd88ad88.gif)

**Comment by TrushantB** [comment]
I am facing the same issue. Please anyone suggest how to fix it.

**Comment by luin** [comment]
The issue should have been fixed in v2.0.0-rc.2 so closing. Feel free to reopen if there are anything unclear.
